nodes/src/nodes/embedding_video/IInstance.py

```
@@ -0,0 +1,223 @@
+    embedding vectors and timestamp metadata.
+    """
+
+    IGlobal: IGlobal
+    """
+    Reference to the global context object of type IGlobal.
+
+    This provides access to shared resources like the embedding model and
+    frame extraction configuration.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)